packages/esm-patient-allergies-app/src/allergies/allergy-intolerance.resource.ts

```diff
@@ -53,7 +53,11 @@ export function useAllergies(patientUuid: string): UseAllergies {
 }
 
 function mapAllergyProperties(allergy: FHIRAllergy): Allergy {
-  const manifestations = allergy?.reaction[0]?.manifestation?.map((coding) => coding.coding[0]?.display);
+  const manifestations = allergy?.reaction[0]?.manifestation?.map((coding) => {
+    const display = coding.coding[0]?.display;
+    const text = coding.text;
+    return display === 'Other' ? text : display;
```

Comment:
This is fragile and unmergable as-is. I _think_ what we actually need here is:

```ts
coding.coding[0]?.display ?? coding.text
```

Reply:
Basically, the string "Other" is a concept name with is locale-specific. If the EMR was in French and the text here was "Autre", this would fail.

Reply:
thank you @ibacher for taking a look at this.
I realised that FHIR AllergyIntolerance response contains a text property which would be better using concepts whose names are language dependent